packages/drivers/odsp-driver/src/odspDeltaStorageService.ts

```diff
@@ -15,7 +15,7 @@ import { TokenFetchOptions } from "./tokenFetch";
 /**
  * Provides access to the underlying delta storage on the server for sharepoint driver.
  */
-export class OdspDeltaStorageService implements api.IDocumentDeltaStorageService {
```

Comment:
why remove the `implement ...`?

Reply:
Because it no longer implements it :) IDocumentDeltaStorageService requires fetchMessages(), and this internal class implements get().